### PR TITLE
docs(website): added documentation for the port forwarding feature

### PR DESCRIPTION
**Pull request by shipsing**
### What does this PR do?
Adds documentation about the port forwarding feature
### Screenshot / video of UI

PR preview: https://67611480e7265db92ed2bd4b--podman-desktop-pr.netlify.app/docs/kubernetes/port-forwarding
### What issues does this PR fix or reference?
https://github.com/podman-desktop/podman-desktop/issues/9673

### How to test this PR?

- [ ] Tests are covering the bug fix or the new feature

### CODE REVIEW

File: website/docs/kubernetes/port-forwarding.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+Within a Kubernetes cluster, you can access an application by its internal IP address. But, if you want to access the application from your local machine, use the port forwarding feature. Using UI, you can forward a local port to a port on the pod that runs your application. This way you can interact with the application running in a Kubernetes cluster from your local machine for debugging and testing purposes.
+
+You can use the port forwarding feature for the pods and services running on a Kubernetes cluster. Also, you can perform port forwarding for any exposed ports.
+
+#### Prerequisites
+
+Make sure you have:
+- A running Podman machine.
+- A running Kubernetes cluster.
```
Comment (axel7083) [inline website/docs/kubernetes/port-forwarding.md]: Might be interesting to link to other part of the documentation such as 

- A running Podman machine. => `/docs/podman/creating-a-podman-machine`
- A running Kubernetes cluster. => `/docs/kind/creating-a-kind-cluster`

**Review by axel7083 — APPROVED** [review]
Re-approving, nice work!

**Review by Firewall — APPROVED** [review]
👍 Awesome

**Comment by cdrage** [comment]
**DO NOT MERGE UNTIL 1.15**

@shipsing @Firewall @axel7083 

As port forwarding features are not yet in the release.

**Comment by benoitf** [comment]
setting to draft until 1.15 is out

**Review by cdrage — APPROVED** [review]

**Pull request merged**